recipes/intel-compilers-repack/meta.yaml

```diff
@@ -0,0 +1,362 @@
+  - url: https://anaconda.org/intel/oneccl-devel/{{ oneccl_version }}/download/{{ target_platform }}/oneccl-devel-{{ oneccl_version }}-intel_{{ oneccl_build_number }}.tar.bz2
+    folder: oneccl-devel
+  {% endif %}
+
+build:
+  number: {{ build_number }}
+  binary_relocation: false
+  detect_binary_files_with_prefix: false
+  skip: True                                  # [not (linux64 or win or osx)]
+
+outputs:
+  - name: intel-cmplr-lic-rt
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    about:
+      home: https://software.intel.com/content/www/us/en/develop/tools.html
+      doc_url: https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html
+      dev_url: https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html
+      summary: Intel End User License Agreement for Developer Tools
+      license: LicenseRef-Proprietary-Intel-End-User-License-Agreement-for-Developer-Tools
+      license_family: Proprietary
+      license_file: 
+        - intel-cmplr-lic-rt/info/licenses/license.txt
+        - intel-cmplr-lic-rt/info/licenses/tpp.txt
+      description: |
+        Intel End User License Agreement for Developer Tools.
+        This package is a repackaged set of binaries obtained directly from Intel's Anaconda.org channel.
+    test:
+      commands:
+        - ls -A1 ${PREFIX}/licensing/compiler/*  # [unix]
+        - dir %PREFIX%\licensing\compiler\*  # [win]
+
+  - name: intel-cmplr-lib-rt
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    build:
+      skip: True  # [osx]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - ocl-icd  # [linux]
+        - zlib
+      run:
+    about:
+      home: https://software.intel.com/content/www/us/en/develop/tools.html
+      doc_url: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/dpc-compiler.html
+      dev_url: https://software.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top.html
+      summary: Runtime for Intel® C++ Compiler Classic
+      license: LicenseRef-Proprietary-Intel-End-User-License-Agreement-for-Developer-Tools
+      license_family: Proprietary
+      license_file: 
+        - intel-cmplr-lib-rt/info/licenses/license.txt
+        - intel-cmplr-lib-rt/info/licenses/tpp.txt
+      description: |
+        Runtime for Intel® C++ Compiler Classic.
+        This package is a repackaged set of binaries obtained directly from Intel's Anaconda.org channel.
+    test:
+      commands:
+        - ls -A1 ${PREFIX}/lib/*  # [unix]
+        - dir %PREFIX%\Library\bin\*  # [win]
+
+  - name: intel-fortran-rt
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    build:                               # [osx]
+      missing_dso_whitelist:             # [osx]
+        - '**/for_mac_ftn_alloc.dylib'   # [osx]
```

Comment:
@napetrov, which package is this dylib supposed to come from?

Reply:
@isuruf  - sorry, missed your question.  Trying to check with compiler team. 
But so far i've found this one - community.intel.com/t5/Intel-Fortran-Compiler/Build-issue-with-2021-3-on-macOS-for-mac-ftn-alloc-dylib/m-p/1344391#M158807 

So excluding warning on missed dso is good workaround for now.

Reply:
This bug have been fixed internally and would be avaialbe in one of the next releases.